openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -499,11 +492,10 @@ const EntityPageInfo = ({
               </div>
             ) : null}
             <Button
-              className="tw-rounded tw-mb-1 tw-flex bg-[#D9CEEE] tw-ml-2"
+              className="tw-rounded tw-mb-1 tw-flex bg-[#D9CEEE] tw-ml-2 tw-border tw-border-primary"
               data-testid="manage-button"
               size="small"
-              theme="primary"
-              variant="outlined"
+              type="default"
```

Comment:
See if type `Primary` eliminate classes or not

Reply:
If we give the primary then it will `contained` button but here we need an `outlined` button.